### ISSUE TITLE
Test suite missing in PyPI tarball

### ISSUE THREAD

**Issue by mgorny** [pyca/service_identity#26]
The 17.0.0 tarball uploaded to PyPI doesn't have the `tests` directory, and it is therefore impossible to test the release. This is important for distro packaging. Could you please fix the `MANIFEST.in` to include tests?

**Comment by sigmavirus24** [comment]
As I understand it, Gentoo has multiple methods for generating a new package, including pulling tarballs from GitHub. As such, it doesn't seem entirely necessary to clutter the installable source package with the tests. Things on PyPI are meant to be pip installable. While some maintainers may choose to include tests, the tests are useless to anyone using the python packaging tooling.

**Comment by mimi1vx** [comment]
@sigmavirus24  standard is download package from pypi..And including tests in sdist helps also for other distributions. 
PIP isn't installable but distribute. Install is only one part of workflow

**Comment by sigmavirus24** [comment]
@mimi1vx It's not just an argument against doing this for Gentoo. The Python packaging guide has two kinds of archives:

- source archive
- source distribution (a.k.a, `sdist`)

An archive, as the name suggests, is the appropriate channel for tests from which one can generate an installable artefact. A distribution is an installable artefact. Just because others have been bullied and confused into including tests in an `sdist` doesn't mean it should actually continue.

That aside, turns out packagers only run the tests when convenient, regardless of whether they're present. I've had packages be shipped with broken patches because the packager didn't actually run the tests that I included at their behest. The same has been happening with other packages I'm aware of.

In short, packagers are overworked and no matter how much I try to help, educate, or guide, it makes little difference in the quality of distro-provided package versions of the software I've seen apart from me directly becoming involved (which I don't have time for).

**Comment by hynek** [comment]
While I agree with Ian, I don't have the energy for this recurring fight so I've just added everything and it will be part of the next release.
